tests/post/steps/test_ui.py

```diff
@@ -13,11 +9,9 @@ def test_ui(host):
 
 
 @then("we can reach the UI")
-def reach_UI(host):
-    ip = utils.get_grain(host, "metalk8s:control_plane_ip")
-
+def reach_UI(host, control_plane_ingress_ep):
     response = requests.get(
-        "https://{ip}:8443".format(ip=ip),
+        control_plane_ingress_ep,
```

Comment:
Realizing this should depend on the value computed by CSC, but to make the test more robust, we could simply find the right host + path from the Ingress object we're testing. Not related to this PR though, I'll open a debt ticket.

Reply:
Hmm right this path is configurable from CSC :thinking:, indeed it will need some update to take this into account, but I will not do it in this PR, so yes if you can create a debt ticket for it :heart:

Reply:
Done in #3421